app/script/util/TimeUtil.js

```diff
@@ -31,9 +31,10 @@ z.util.TimeUtil = {
   /**
    * Format milliseconds into 15s, 2m.
    * @param {number} duration - Duration to format in milliseconds
+   * @param {number} maximumUnits - Maximum number of units shown in the textual representation
    * @returns {Object} Unit, value and localized string
    */
-  formatDuration: duration => {
+  formatDuration: (duration, maximumUnits = 1) => {
```

Comment:
Where else did we use this? Can we safely set this default value? I suppose it is ok, because it only affects the text which is set in one place only.

Reply:
I wrote this method to replace some scattered methods which did all pretty much the same. I know where and how this is used and can assure that the default value is save.